### PR TITLE
[cinder-csi-plugin] Enable ephemeral e2e tests 

### PR DESCRIPTION
**Pull request by ramineni**
**What this PR does / why we need it**:
This PR enables e2e tests for ephemeral volumes.

**Which issue this PR fixes(if applicable)**:
fixes #

**Special notes for reviewers**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by ramineni** [comment]
@lingxiankong @jichenjc Ready for review. PTAL

File: manifests/cinder-csi-plugin/cinder-csi-nodeplugin.yaml [hunk 1]
```diff
@@ -63,6 +63,7 @@ spec:
             - "--nodeid=$(NODE_ID)"
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--cloud-config=$(CLOUD_CONFIG)"
+            - "--v=5"
```
Comment (jichenjc) [inline manifests/cinder-csi-plugin/cinder-csi-nodeplugin.yaml:66]: previously , I was told in different PR this should not be set to verbose by default as user always has capability to do so by default we should not set log level..

Comment (ramineni) [reply manifests/cinder-csi-plugin/cinder-csi-nodeplugin.yaml:66]: @jichenjc Thanks for pointing out. I just added to debug the failures before. Missed to remove it. Ill update

**Comment by ramineni** [comment]
@jichenjc updated. PTAL

**Comment by jichenjc** [comment]
/lgtm
/approve

thanks for the update, we have lgtm before and there is only minor change

**Pull request merged**